helix-term/src/commands.rs

```diff
@@ -335,6 +336,8 @@ impl MappableCommand {
         goto_last_diag, "Goto last diagnostic",
         goto_next_diag, "Goto next diagnostic",
         goto_prev_diag, "Goto previous diagnostic",
+        goto_next_error, "Goto next error diagnostic",
+        goto_prev_error, "Goto prev error diagnostic",
```

Comment:
```suggestion
        goto_next_error, "Goto next error",
        goto_prev_error, "Goto prev error",
```
I guess error should be enough?

Reply:
Thx! Ill update the pr with the suggestions when i have time